.github/workflows/native-cron-build.yml

```diff
@@ -21,10 +21,21 @@ jobs:
       - name: Pull docker image
         run: docker pull quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }}
 
-      - name: Set up Java
-        uses: actions/setup-java@v1
+      - name: Set up JDK ${{ matrix.java-version }}
```

Comment:
I don't see `java-version` defined in the matrix?

Reply:
Damn it, I hadn't changed it everywhere. Should be good now.